client/landing/jetpack-cloud/components/threat-description/index.jsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class ThreatDescription extends Component {
+	static propTypes = {
+		threat: PropTypes.object,
```

Comment:
Inseated of passing the whole threat object can we pass in each individual props?